indra/newview/llreflectionmapmanager.cpp

```diff
@@ -906,6 +906,8 @@ void LLReflectionMapManager::updateUniforms()
         // the box probe
         LLMatrix4 refBox[LL_MAX_REFLECTION_PROBE_COUNT];
 
+        LLMatrix4 heroBox;
```

Comment:
Just a random thought. iirc LLMatrix4 uses OpenGL matrix conventions, but there is something different about the matrix calculations (floating point precision-related, possibly). I wonder if there is value in a convention of using glm data types when we know the data is about to be passed directly to the shader. I don't have a compelling use case for that at this time, though.